tasks/server-tests/bothServer.test.mts

```diff
@@ -1,191 +1,170 @@
-import { fileURLToPath } from 'node:url'
-
-import { afterAll, afterEach, beforeAll, describe, expect, it } from 'vitest'
-import { fs, path, $ } from 'zx'
-
-import { getConfig } from '@redwoodjs/project-config'
-import { sleep } from './helpers.mjs'
-
-//////////
-// Setup
-//////////
-
-$.verbose = !!process.env.VERBOSE
-
-const __dirname = fileURLToPath(new URL('./', import.meta.url))
-const FIXTURE_PATH = fileURLToPath(
-  new URL('./fixtures/redwood-app', import.meta.url)
-)
-
-// @redwoodjs/cli (yarn rw)
-const rw = path.resolve(__dirname, '../../packages/cli/dist/index.js')
-// @redwoodjs/api-server (yarn rw-server)
-const rwServer = path.resolve(
-  __dirname,
-  '../../packages/api-server/dist/bin.js'
-)
-// @redwoodjs/web-server (yarn rw-web-server)
-const rwWebServer = path.resolve(
-  __dirname,
-  '../../packages/web-server/dist/bin.js'
-)
-
-let original_RWJS_CWD
-let projectConfig
-beforeAll(() => {
-  original_RWJS_CWD = process.env.RWJS_CWD
-  process.env.RWJS_CWD = FIXTURE_PATH
-  projectConfig = getConfig()
-  console.log([
-    'These tests use the following command to run the server:',
-    `• RWJS_CWD=${process.env.RWJS_CWD} yarn node ${rw} serve`,
-    `• RWJS_CWD=${process.env.RWJS_CWD} yarn node ${rwServer}`,
-    `• RWJS_CWD=${process.env.RWJS_CWD} yarn node ${rwWebServer}`,
-  ].join('\n'))
-})
-afterAll(() => {
-  process.env.RWJS_CWD = original_RWJS_CWD
-})
-
-// Clean up the child process after each test
-let p
-afterEach(async () => {
-  if (!p) {
-    return
-  }
-  p.kill()
-  // Wait for child process to terminate
-  try {
-    await p
-  } catch {
-    // Ignore
-  }
-})
+import { describe, expect, it } from 'vitest'
+import { $ } from 'zx'
 
-const TIMEOUT = 1_000 * 2
+import { rw, rwServer } from './vitest.setup.mjs'
 
-//////////
-// Tests
-//////////
-
-describe.each([
-  [[rw, 'serve']],
-  [rwServer],
-])('serve both (%s)', (cmd) => {
+describe('rw', () => {
```

Comment:
```suggestion
describe('rw serve', () => {
```

Reply:
Alright, so, yeah, either do `describe('rw serve', ...` here. Or change the `it` below to ``it("has help configured for `serve`", ...``
And something similar for other `it`s